examples/nextjs/.env.example

```
@@ -1,7 +1,7 @@
 # Nacelle
 NEXT_PUBLIC_NACELLE_SPACE_ID=<your-nacelle-space-id>
 NEXT_PUBLIC_NACELLE_GRAPHQL_TOKEN=<your-nacelle-space-token>
-NEXT_PUBLIC_NACELLE_ENDPOINT=<nacelle-storefront-graphql-endpoint> # e.g. https://hailfrequency.com/v2/graphql
+NEXT_PUBLIC_NACELLE_ENDPOINT=<nacelle-storefront-graphql-endpoint>
```

Since the advice is usually to copy/paste the .env.example into .env could we set people up with the default here? Since it seems like most people will be using the default url? 

```suggestion
NEXT_PUBLIC_NACELLE_ENDPOINT=https://hailfrequency.com/v2/graphql
```

Obviously this might change if we start giving merchants unique subdomains or something. But it'd probably work most of the time. 

Long term we _should_ add comments above each .env variable to explain what it is like we do for the Shopify checkout token stuff.

---

> Since the advice is usually to copy/paste the .env.example into .env could we set people up with the default here? Since it seems like most people will be using the default URL?

Sure! Changed to the v3 Hail Frequency endpoint in https://github.com/getnacelle/nacelle-react/pull/67/commits/6ac981ee0f911a2aade0792a41b2e4dff14bcb62. We'll need to update that value when we add support for Warp 2, but that's okay.

> Long term we should add comments above each .env variable to explain what it is like we do for the Shopify checkout token stuff.

Yeah. And as the number of possible CMS vendors increases, we'll probably need to shift to a different approach altogether, to avoid `.env.example` and other docs from increasing in complexity. It feels like a good opportunity to take advantage of Nacelle Docs 2.0's stack adaptiveness, and provide tool for creating a stack-specific `.env` for our various starters.